commands/cluster/cluster.go

```diff
@@ -34,3 +39,12 @@ func init() {
 	ClusterCmd.AddCommand(clusterShutdownCmd)
 	ClusterCmd.AddCommand(clusterVersionCmd)
 }
+
+func isCloudInvocation(cmd *cobra.Command) bool {
+	token, _ := cmd.InheritedFlags().GetString("cloud-token")
+	if token != "" {
+		fmt.Println("Hazelcast clc does not support cluster operations on cloud. Go checkout https://github.com/hazelcast/hazelcast-cloud-cli for cluster management on cloud.")
```

Comment:
How about: "Cluster operations on Cloud are not supported. Checkout https://github.com/hazelcast/hazelcast-cloud-cli for cluster management on Cloud."

Reply:
Note that this function would be more valuable with just returning whether a command is called with Cloud parameters. So, the caller of this function can decide what to do (print a warning in this case).

The function then could be simplifed to `return token != ""`.

Reply:
If you'd like to keep the side effect, it would be better to rename the function to something like: `ensureNotCloud`.

Reply:
I hesitated while doing this. Side effect removes log line duplication but I think it was not the right way to go. Lets do as you suggested. I will fix this